packages/platforms/inventec/x86-64/x86-64-inventec-d7032q28b/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -30,9 +30,13 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <AIM/aim.h>
+#include <onlplib/file.h>
+#include <onlp/onlp.h>
 #include "platform_lib.h"
 
-int deviceNodeWrite(char *filename, char *buffer, int buf_size, int data_len)
+#define PSU_NODE_MAX_PATH_LEN 64
+
+int _onlp_file_write(char *filename, char *buffer, int buf_size, int data_len)
```

Comment:
You don't need all of the onlp_file_{write,read} implementations. Perhaps my previous comments were too brief. 

Please use the exisiting ones already available in the onlplib module:
https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/base/any/onlp/src/onlplib/module/inc/onlplib/file.h